spec.html

```diff
@@ -37634,12 +37549,86 @@ <h1>
             1. Let _ySmaller_ be <emu-meta suppress-effects="user-code">IsLessThan(_yString_, _xString_, *true*)</emu-meta>.
             1. If _ySmaller_ is *true*, return *1*<sub>𝔽</sub>.
             1. Return *+0*<sub>𝔽</sub>.
+          1. Return ? SortIndexedProperties(_obj_, _len_, _SortCompare_).
+        </emu-alg>
+        <emu-note>
+          <p>Method calls performed by the ToString abstract operations in steps <emu-xref href="#step-sortcompare-tostring-x"></emu-xref> and <emu-xref href="#step-sortcompare-tostring-y"></emu-xref> have the potential to cause _SortCompare_ to not behave as a consistent comparator.</p>
+        </emu-note>
+        <emu-note>
+          <p>Because non-existent property values always compare greater than *undefined* property values, and *undefined* always compares greater than any other value, *undefined* property values always sort to the end of the result, followed by non-existent property values.</p>
```

Comment:
I note that this swaps the order of this note and the previous one. I guess I don't really care either way, but was there a reason for that change?

Reply:
In this morning's version, the A.p.sort clause still had the 3 bullets, the last of which talked about applications of ToString possibly producing inconsistent results, so I thought it made sense to follow that with the note about ToString possibly calling methods.

But now that the 3-bullet list is gone, I could unswap them.